### ISSUE TITLE
Standardize Job Status

### ISSUE THREAD

**Issue by thiagopnts** [NYTimes/video-captions-api#57]
right now its just a string and we just use whatever comes from the Provider with a few of our own. We should standardize this, and have a stable list of possible status for each Job

**Comment by markx** [comment]
Won't do.

**Comment by thiagopnts** [comment]
@markx why we don't want this anymore?

**Comment by markx** [comment]
@thiagopnts  Because we decided to do it on the MF API, right?

**Comment by thiagopnts** [comment]
@markx yes, for media factory<->scoop communication. This one is for the captions API only, having something standard here makes it easier to handle job status, better docs etc
